bedrock/base/templates/macros.html

```diff
@@ -266,23 +266,34 @@ <h3 class="mzp-c-sendyourself-title">
         </div>
         <div class="mzp-c-sendyourself-formfields">
           <p class="mzp-c-sendyourself-field">
-            <label for="{{ id }}-input" class="mzp-js-input-label" data-alt="{{ _('Enter your email or phone number') }}">{{ _('Enter your email') }}</label>
+            <label for="{{ id }}-input" class="mzp-js-input-label" data-alt="{{ _('Enter your email or phone number') }}">
+              {% if field_title %}
+                {{ field_title }}
+              {% else %}
+                {{ _('Enter your email') }}
+              {% endif %}
+            </label>
             <input id="{{ id }}-input" class="mzp-c-sendyourself-input" name="phone-or-email" type="text" required>
-            <button type="submit" class="mzp-c-button">{{ _('Send') }}</button>
+            <button type="submit" class="mzp-c-button {{ button_theme }}">{{ _('Send') }}</button>
           </p>
           <p class="mzp-c-sendyourself-note mzp-is-sms">
             {{ _('SMS service available in select countries only. SMS &amp; data rates may apply.') }}
             {{ _('The intended recipient of the email or SMS must have consented.') }} <a href="{{ url('privacy.notices.websites') }}#campaigns">{{ _('Learn more') }}</a>
           </p>
           <p class="mzp-c-sendyourself-note mzp-is-email">
-            {{ _('The intended recipient of the email must have consented.') }} <a href="{{ url('privacy.notices.websites') }}#campaigns">{{ _('Learn more') }}</a>
+            {% if field_note %}
+              {{ field_note }}
+            {% else %}
+              {{ _('The intended recipient of the email must have consented.') }}
+            {% endif %}
+            <a href="{{ url('privacy.notices.websites') }}#campaigns">{{ _('Learn more') }}</a>
           </p>
         </div>
         <div class="mzp-c-sendyourself-thanks mzp-u-hidden">
           <h3>{{ _('Your download link was sent.') }}</h3>
           <p class="mzp-is-sms">{{ _('Check your device for the email or text message!') }}</p>
           <p class="mzp-is-email">{{ _('Check your device for the email!') }}</p>
-          <button class="button button-hollow button-light mzp-js-send-another" type="button">{{ _('Send to another device') }}</button>
+          <button class="mzp-c-button mzp-t-secondary mzp-t-small mzp-t-dark mzp-js-send-another" type="button">{{ _('Send to another device') }}</button>
```

Comment:
Sorry missed this initially, but this causes a contrast issue on the 62 whatsnew page 😕

<img width="366" alt="screenshot 2019-01-17 at 20 06 37" src="https://user-images.githubusercontent.com/400117/51345697-86777080-1a93-11e9-8cff-d9ef672e8a29.png">